kernel/kernel-auth-adapter/src/main/java/io/mosip/kernel/auth/defaultadapter/constant/AuthAdapterConstant.java

```diff
@@ -42,5 +30,21 @@
 	public static final String HTTP_METHOD_NOT_NULL = "Http Method Cannot Be Null";
 
 	public static final String ROLES_NOT_EMPTY_NULL = "Roles Cannot Be Empty or Null";
+	
+	public final static String APPTYPE_UIN = "UIN";
+
+	public final static String APPTYPE_USERID = "USERID";
+
+	public final static String APPTYPE_USER = "USERIDTYPE";
+
+	public final static String USERPWD_SUCCESS_MESSAGE = "Username and password combination had been validated successfully";
+
+	public final static String CLIENT_SECRET_SUCCESS_MESSAGE = "Clientid and Token combination had been validated successfully";
+
+	public final static String TOKEN_SUCCESS_MESSAGE = "Token had been validated successfully";
+
+	public static final String AUTH_COOOKIE_HEADER = "Authorization";
```

Comment:
What is the difference betweeen this and AUTH_REQUEST_COOOKIE_HEADER ?

Reply:
removed